From 207c10aff76176e559b858402abb06617948eb54 Mon Sep 17 00:00:00 2001
From: Tien Hock Loh <thloh@altera.com>
Date: Thu, 3 Dec 2015 15:11:53 +0800
Subject: [PATCH 46/73] FogBugz #338696: Update socfpga_defconfig for PCI-E
 reference design

This is to add PCI-E drivers and tested peripherals to
socfpga_defconfig

Signed-off-by: Tien Hock Loh <thloh@altera.com>
[mli1:Original patch taken from
https://github.com/altera-opensource/linux-socfpga.git socfpga-4.1-ltsi]
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/configs/socfpga_defconfig |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/socfpga_defconfig b/arch/arm/configs/socfpga_defconfig
index f7016f7..10e5e07 100644
--- a/arch/arm/configs/socfpga_defconfig
+++ b/arch/arm/configs/socfpga_defconfig
@@ -155,3 +155,10 @@ CONFIG_DETECT_HUNG_TASK=y
 CONFIG_FUNCTION_TRACER=y
 CONFIG_DEBUG_USER=y
 CONFIG_XZ_DEC=y
+CONFIG_PCI=y
+CONFIG_PCI_MSI=y
+CONFIG_PCIE_ALTERA=y
+CONFIG_PCIE_ALTERA_MSI=y
+CONFIG_E1000E=m
+CONFIG_BLK_DEV_NVME=m
+CONFIG_IGB=m
-- 
1.7.5.4

